odoo/addons/base/models/res_users.py

```diff
@@ -931,12 +931,6 @@ def _register_hook(self):
         if hasattr(self, 'check_credentials'):
             _logger.warning("The check_credentials method of res.users has been renamed _check_credentials. One of your installed modules defines one, but it will not be called anymore.")
 
-    def _get_placeholder_filename(self, field=None):
```

Comment:
it is most than probably used by website (via the binary_content controller) to display the user placeholder.

And just to give my 2 cents anyway, now that I comment btw :arrow_up: :
About the PR, don't you think that you will have a strange layout mixin kanban with image and withtout image ? 
Probably that keep img tag will help you to easily keep a consistent design, even if display is set in hidden, or placeholder as a transparent image.

Reply:
ok thanks, I'll reintroduce the user placeholder.
About mixing image/non-images kanban card, are you concerned about the DOM structure consistency or more the UI consistency ?

Reply:
It was more about the dom consistency. If you hide the picture or display a transparent placeholder, the UI will be btw 'not' consistent, but it is expected and by design.

But I already see future side effect, with some img:after { margin-right:10px } or similar rules  that will make the UI really inconsistent with wrong aligment and co.

Reply:
Well I agree it has implications on css rules.
But IMHO, a rule such as `img:after { margin-right:10px }` is really specific to an image. If there is no visible image, I would expect the rule to be not applied.
And if it should nonetheless be applied, then it's propably the css selector which should be changed.
I think the current solution in this PR is the most "natural" way of doing it and allows to be more flexible... at the cost of unwanted design inconsistencies possibly introduced by sleepy devs.
I guess it's a matter of choice ^^